Docs/simplify index pages (#3995)
514-labs/moose: apps/framework-docs-v2/content/moosestack/streaming/schema-registry.mdx

---
title: Schema registry

> description: Reference for using Confluent Schema Registry with Moose streams

Reviewer: PV

order: 7
category: streaming
---

import { Callout, LanguageTabs, LanguageTabContent } from "@/components/mdx";

# Schema registry

<Callout type="info" title="Current support">
Only JSON Schema is supported today. Avro and Protobuf are planned.
</Callout>

> Use `schemaConfig` in TypeScript or `schema_config` in Python when a [Stream](/moosestack/streaming/create-stream) should publish through Confluent Schema Registry.

Reviewer: ```suggestion
Use schemaConfig (TypeScript) or schema_config (Python) to publish a [Stream](/moosestack/streaming/create-stream) through Confluent Schema Registry.
```

  </LanguageTabContent>
</LanguageTabs>

> ### Kind field

Reviewer: ```suggestion
### Schema Registry "Kind"
```

> ### Kind field

Reviewer: Seems like this and the below should be an API reference kind of business?

Reviewer: They are a whole section about one variable?

Author: Same framing here: I want this page-level reference to own the important `schemaConfig` / `schema_config` details as the separate API reference goes away. I compressed the thin `kind` / `reference` sections into one tighter config-fields block, but I’m keeping the field-level guidance on-page instead of deferring it elsewhere.
